src/main/webapp/src/components/entityAnalysisInfo.js

```diff
@@ -99,7 +99,7 @@ class EntityAnalysisInfo extends React.Component {
     function getTooltip(entity) {
       return '<div style="font-size:15px">' + entity.name + '<br>' 
           + '<b>Type: </b>' + entity.type + '<br>' 
-          + '<b>Salience: </b>' + entity.salience + '<br>' 
+          + '<b>Salience: </b>' + entity.salience.toFixed(2) + '<br>' 
           + ((entity.wikiLink !== "") ? 'Click for wiki link' : '')
           + '</div>';
     }
```

Comment:
on line 135 you haven't changed `entity.salience` to `entity.salience.toFixed(2)`. BUT it should be `Number(entity.salience.toFixed(2))`, because on line 135 it's important to have a number here, since charts use it do define section sizes of the pies